lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+      {% if site.twitter.smallSummary %}
```

Comment:
Yep! You can chain defaults, e.g., `| default: page.twitter.card | default: site.twitter.card...`